src/backend/distributed/metadata/metadata_cache.c

```diff
@@ -1967,6 +1970,26 @@ CitusHasBeenLoadedInternal(void)
 }
 
 
+/*
+ * GetCitusCreationLevel returns the level of the transaction creating citus
+ */
+int
+GetCitusCreationLevel(void)
+{
+	return CreateCitusTransactionLevel;
+}
+
+
+/*
+ * Sets the value of CreateCitusTransactionLevel based on int received
```

Comment:
maybe: "based on int received" -> "which represents the nesting level of the subtransaction that created the Citus extension."